test/WebJobs.Script.Tests/EndToEndTestFixture.cs

```diff
@@ -38,6 +37,17 @@ protected EndToEndTestFixture(string rootPath)
 
         public ScriptHost Host { get; private set; }
 
+        public CloudQueue GetNewQueue(string queueName)
+        {
+            string connectionString = AmbientConnectionStringProvider.Instance.GetConnectionString(ConnectionStringNames.Storage);
+            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(connectionString);
+            CloudQueueClient queueClient = storageAccount.CreateCloudQueueClient();
+            var queue = queueClient.GetQueueReference(queueName);
+            queue.CreateIfNotExists();
+            queue.Clear();
+            return queue;
+        }
+
         private void CreateTestStorageEntities()
         {
             string connectionString = AmbientConnectionStringProvider.Instance.GetConnectionString(ConnectionStringNames.Storage);
```

Comment:
Recommend factoring the existing test code a bit so we only read the connection/storage account once, and the fixture class exposes a CloudQueueClient/CloudBlobClient members. These GetNewQueue/CreateStorageEntities methods will then have less duplicated code.